docs/ota-client-guide/modules/ROOT/pages/evaluation-to-prod.adoc

```diff
@@ -23,11 +23,11 @@ No matter which phase you're in, there are 4 basic tasks you'll need to do:
 * Provision devices with authentication credentials for your account
 * Send updated images to some or all of your devices
 
-This guide contains chapters to guide you through each phase and the following sections give you an introduction the phased approach:
+This guide contains chapters to guide you through each phase, and the following sections give you an introduction to the phased approach:
 
 == Evaluate OTA Connect
 
-During evaluation, you should focus on testing the basic update functionality to make sure that you understand how it works. At this stage you don't need to think about customization or production-level security. You'll use basic, minimal device images for test boards, or even just simulate devices, and OTA Connect will handle the software signing on the server. For provisioning, you'll use a shared registration key for all your devices.
+During the evaluation phase, you should focus on testing the basic update functionality to make sure that you understand how it works. At this stage, you don't need to think about customization or production-level security. You'll use basic, minimal device images for test boards, or even just simulate devices, and OTA Connect will handle the software signing on the server. For provisioning, you'll use a shared registration key for all your devices.
```

Comment:
```suggestion
During the evaluation phase, test the basic update functionality to make sure that you understand how it works. At this stage, you don't need to think about customization or production-level security. Use basic, minimal device images for test boards, or even just simulate devices, and OTA Connect will handle the software signing on the server. For provisioning, use a shared registration key for all your devices.
```
If we can, let's omit the future tense in instructions and use the imperative mood. Applies also to the sections below.

Reply:
The imperative mood changes the tone of the doc to become formal but as Jon informed me, the tone of the developer guide should be informal. Thus imperative tones should be avoided when possible.